bin/out

```diff
@@ -10,6 +10,16 @@ export AWS_ACCESS_KEY_ID=$(jq -r '.source.aws_access_key_id // empty' < /tmp/inp
 export AWS_SECRET_ACCESS_KEY=$(jq -r '.source.aws_secret_access_key // empty' < /tmp/input)
 export AWS_DEFAULT_REGION=$(jq -r '.source.region // empty' < /tmp/input)
 
+# remove any empty credentials vars so the AWS client will try instance profiles
+# https://www.packer.io/docs/builders/amazon.html#using-an-iam-instance-profile
+if [ -z "$AWS_ACCESS_KEY_ID" ]; then
+  unset AWS_ACCESS_KEY_ID
+fi
```

Comment:
Heh, I like this better than the https://github.com/jdub/ami-resource/pull/3 approach.